Add documentation for object store layer. (#418)
heremaps/here-data-sdk-typescript: docs/using-object-store-layer.md

      "blob", 
      "v2", 
      settings, 
      HRN.fromString("your-catalog-hrn")
   );
   ```

## Get data from an object store layer

You can request any data from an [object store layer](https://developer.here.com/documentation/data-user-guide/user_guide/portal/layers/layers.html#object-store-layers) using a key or datahandle.

**To get data from the object store layer:**

1. Create the `RequestBuilder` object.

   For instructions, see [Create RequestBuilder](#use-objectstoreapi-from-hereolp-sdk-dataservice-api).

2. Call the `getBlobByKey` function with the `key` and `layerId` parameters.

   ```typescript
   const result = await ObjectStoreApi.getBlobByKey(requestBuilder, {
      key: "your-data-key",
      layerId: "your-layer-id"
   });
   ```

You receive data from the requested key of the selected layer.

## Get all keys

1. Create the `RequestBuilder` object.

   For instructions, see [Create RequestBuilder](#use-objectstoreapi-from-hereolp-sdk-dataservice-api).

2. Call the `listKeys` function with the `layerId` parameter.

   ```typescript
   const result = await ObjectStoreApi.listKeys(requestBuilder, {
      layerId: "your-layer-id"
   });
   ```

> You receive a list of all keys from the selected layer for one level.
> The list can be like virtual directory-like structure created from keys in the 
> storage where '/' is used as a separator.
> The response will contain for each key an indication if it is a folder or object. For each folder, 
> the we must recursively call this function with a parent in the query string 
> to retrieve the next level of keys for the folder.

Reviewer: ```suggestion
You receive a list of all keys from one level of the selected layer.
The list is a virtual directory. Its structure consists of keys in
the storage where slashes (/) are used as separators.
For each key, the response indicates if it is a folder or object.
To retrieve the next level of keys in a folder,
call this function recursively with a parent in the query string.
```
